packages/neuron-wallet/src/services/ckb-runner.ts

```diff
@@ -142,18 +142,24 @@ export const startCkbNode = async () => {
     }
   })
 
-  ckb.on('error', error => {
+  currentProcess.on('error', error => {
     logger.error('CKB:\trun fail:', error)
     isLookingValidTarget = false
-    ckb = null
+    if (Object.is(ckb, currentProcess)) {
+      ckb = null
```

Comment:
I think `stop signal` should be sent to process `ckb` even if it has been sent, because the reason `ckb` and `currentProcess` conflict here is unknown.

Reply:
Setting the `ckb` as null can not stop the process, I guess adding a condition to set it as null have no side effect.

Reply:
Ah, I found I recognized the condition as `!Object.is(ckb, currentProcess)` mistakenly. Now I have no questions